pages/api/user/index.ts

```diff
@@ -19,7 +25,26 @@ export default async function handle(
             break;
         }
         case "POST": {
-            // TODO:
+            let newUser;
+            const userRole = req.body.role;
+            // TODO (6/27/21): define these roles as an enum somewhere? but where?
```

Comment:
good idea - models maybe?

Reply:
Yeah, we need to use roles in the NextAuth call back too

Reply:
the schema.prisma file contains enums for locales and provinces - should we just refer to those? or should they be defined again in a separate file?

Reply:
@jason17h Are you talking about roles?